crates/store/re_redap_client/src/grpc.rs

```diff
@@ -444,6 +373,91 @@ pub async fn stream_blueprint_and_partition_from_server(
     Ok(())
 }
 
+pub type FetchChunksResponseStream = std::pin::Pin<
+    Box<
+        dyn Stream<Item = Result<re_protos::cloud::v1alpha1::FetchChunksResponse, tonic::Status>>
+            + Send,
+    >,
+>;
+
+/// Fetches all chunks for a specified partition. You can include/exclude static/temporal chunks.
+pub async fn fetch_partition_chunks(
+    client: &mut ConnectionClient,
+    dataset_id: EntryId,
+    partition_id: PartitionId,
+    exclude_static_data: bool,
+    exclude_temporal_data: bool,
+    query: Option<re_protos::cloud::v1alpha1::Query>,
+) -> Result<FetchChunksResponseStream, StreamError> {
```

Comment:
This entire function could be a `ConnectionClient` method. That would count as a win w.r.t its use of `inner` :)

Ideally it would be broken into its query and fetch parts for when that's useful, but it would already be nice to be left as a TODO.


```suggestion
pub async fn fetch_partition_chunks(
    &mut self,
    dataset_id: EntryId,
    partition_id: PartitionId,
    exclude_static_data: bool,
    exclude_temporal_data: bool,
    query: Option<re_protos::cloud::v1alpha1::Query>,
) -> Result<FetchChunksResponseStream, StreamError> {
```

Reply:
yeah, agree :) let me make that change.